ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -181,6 +181,20 @@ pipeline {
             }
         } } }
 
+        stage('Addon update Check') { steps { script {
```

Comment:
doesnt this stage need to be **after** we clone the repo?

Reply:
xD hahaha that is why it is failing

Reply:
Like a noob